torch/csrc/utils/tensor_new.cpp

```diff
@@ -225,6 +225,14 @@ void recursive_store(char* data, IntArrayRef sizes, IntArrayRef strides, int64_t
 
   PyObject** items = PySequence_Fast_ITEMS(seq.get());
   for (int64_t i = 0; i < n; i++) {
+#ifdef USE_NUMPY
+    if (PyArray_Check(items[i])) {
+      TORCH_WARN(
```

Comment:
TOCH_WARN_ONCE (see https://github.com/pytorch/pytorch/wiki/Developer-FAQ#when-should-i-torch_warn-vs-torch_warn_once)